### PR TITLE
Feature es6

### PR DESCRIPTION
**Pull request by Graciano1997**
In this Project, I did:

Used ES6 modules to write modular JavaScript.
Imported luxon module to display current date and Time
Used npm as software package system.
Deployed the awesome-book project
Update Readme file
Fix linter errors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,161 @@
...
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Portfolio] <a name="about-project"></a>
```
Comment (julie-ify) [inline README.md]: - [ ] Kindly rename the project name.

File: README.md [hunk 2]
```diff
@@ -1 +1,161 @@
...
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>Awesome books: with ES6</b></h3>
+</div>
+
+# 📗 Table of Contents
+
+- [� Table of Contents](#-table-of-contents)
+- [📖 \[Portfolio\] ](#-portfolio-)
```
Comment (julie-ify) [inline README.md]: - [ ] Kindly rename the project name link.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @Graciano1997,

Good job so far!

## Highlights!
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by V-Blaze — APPROVED** [review]
Hi @Graciano1997  , 

# Approved 🟢 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy

## Highlights

- Used ES6 syntax💯
-  Correct Github flow👍
-  LInters setup correctly🟢 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**